hl/src/H5LTanalyze.c

```diff
@@ -2413,11 +2413,7 @@ yy_scan_buffer(char *base, yy_size_t size)
  *       yy_scan_bytes() instead.
  */
 YY_BUFFER_STATE
-yy_scan_string(const char *yystr)
-{
-
-    return yy_scan_bytes(yystr, (int)strlen(yystr));
-}
+yy_scan_string(const char *yystr) { return yy_scan_bytes(yystr, (int)strlen(yystr)); }
```

Comment:
Can you update the format_source script to skip this file, it is in the github action but our format script.

Reply:
OK, I'll take care of that later today.